app/models/MovementListSearchOptions.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import models.MovementListSearchOptions.{DEFAULT_INDEX, DEFAULT_MAX_ROWS}
+import models.MovementSortingSelectOption.ArcAscending
+import play.api.mvc.QueryStringBindable
+
+case class MovementListSearchOptions(sortBy: MovementSortingSelectOption = ArcAscending,
+                                     index: Int = DEFAULT_INDEX,
+                                     maxRows: Int = DEFAULT_MAX_ROWS) {
+
+  val startingPosition: Int = ((index-1) * maxRows) + 1
```

Comment:
what happens if the index is zero, would this work as a negative number?

Reply:
There's a guard against this in the controller